projects/wave-core-new/src/lib/operators/dialogs/expression-operator/expression-operator.component.ts

```diff
@@ -211,9 +211,7 @@ export class ExpressionOperatorComponent implements AfterViewInit, OnDestroy {
                     params: {
                         expression,
                         output_type: dataType.getCode(),
-                        output_no_data_value: {
-                            U8: 42 // TODO: make this configurable once units exist again
-                        }
+                        output_no_data_value: dataType.no_data(42),  // TODO: make this configurable once units exist again
```

Comment:
maybe use datatype MAX or MAX for NODATA in the meantime?

Reply:
0365c2d4a4bb46141542e9a67e1c4f961c2e097a